### PR TITLE
Update e2e_test script to run test packages concurrently in CD pipeline

### PR DESCRIPTION
**Pull request by vipnydav**
### Description
Updated e2e_test.sh to run test packages concurrently in the CD pipeline

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Used louhi pipeline for testing this script on 16 VMs.
2. Unit tests - NA
3. Integration tests - Used this script to run integration tests on 16 VMs in louhi pipeline.

### CODE REVIEW

File: tools/cd_scripts/e2e_test.sh [hunk 1]
```diff
@@ -139,28 +140,192 @@ git checkout $(sed -n 2p ~/details.txt) |& tee -a ~/logs.txt
...
+    exit_code_parallel=$?
+    if [ $exit_code_parallel != 0 ]; then
+      exit_code=$exit_code_parallel
+    fi
+  done
+  return $exit_code
+}
+
+function run_e2e_tests_for_flat_bucket() {
+  flat_bucket_name_non_parallel=$(sed -n 3p ~/details.txt)
```
Comment (Tulsishah) [inline tools/cd_scripts/e2e_test.sh:226]: nit: Add echo for "Running tests for FLAT bucket". same for HNS

**Review by Tulsishah — DISMISSED** [review]
Minor comment.

**Review by Tulsishah — COMMENTED** [review]
Write testing details that you have tested on louhi with 16 VMs.

File: tools/cd_scripts/e2e_test.sh (hunk 1 above)
Comment (vipnydav) [reply tools/cd_scripts/e2e_test.sh:226]: Added.

**Review by Tulsishah — APPROVED** [review]

**Review by Tulsishah — APPROVED** [review]

**Pull request merged**